research/conditional/utils/check_args.py

```diff
@@ -38,3 +38,8 @@ def check_args(args):
 
     if args.repeater_mode:
         assert args.relative_init_scale == None
+
+    if args.n_training_tokens_in_billions is not None:
+        assert (
+            args.n_steps is None
```

Comment:
not sure if I understand it correctly, but since n_steps is required it has to be set to None explicitly via yaml. There was a thread recently on Discord on how to do it, but I don't think that it's the intended feature, or is it?

Reply:
right, I set default to None. Nonetheless, it might still be necessary to set `n_steps: null` explicitly in yaml - we often derive some default n_steps from yaml configs (see for example `configs/baselines/common.yaml`)

Reply:
ok, makes sense